tests/test_exec_250.py

```diff
@@ -0,0 +1,116 @@
+#
+#    Copyright (C) 2010-2017 PyTRiP98 Developers.
+#
+#    This file is part of PyTRiP98.
+#
+#    PyTRiP98 is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    PyTRiP98 is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with PyTRiP98.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+TODO: documentation here.
+"""
+import os
+import unittest
+import logging
+
+import pytrip as pt
+import pytrip.tripexecuter as pte
+
+import tests.base
+
+logger = logging.getLogger(__name__)
+
+
+class TestLocalExec250(unittest.TestCase):
+    """
+    Tests for pytrip.tripexecuter, using new API slated for 2.5.0
+    """
+    def setUp(self):
+        """ Prepare test environment.
+        """
+        testdir = tests.base.get_files()
+
+        self.patient_name = "tst003000"
+
+        self.ctx_path = os.path.join(testdir, self.patient_name + '.ctx')
+        self.vdx_path = os.path.join(testdir, self.patient_name + '.vdx')
+        self.trip_path = os.path.join(testdir, "TRiP98")
+
+    def test_exec(self):
+        """ Prepare and execute a dry-run test using the Executer.
+        """
+        logger.info("Test norun TRiP98 execution")
+
+        logger.debug("Load CtxCube {:s}".format(self.ctx_path))
+        c = pt.CtxCube()
+        c.read(self.ctx_path)
+
+        logger.debug("Load VdxCube {:s}".format(self.vdx_path))
+        v = pt.VdxCube(c)
+        v.read(self.vdx_path)
+
+        print(v.voi_names())
+
+        kernel = pte.KernelModel("C12 Ions RiFi 3MM")
+        kernel.projectile = pte.Projectile("C")
+        kernel.ddd_path = "~/TRiP98/base/DATA/DDD/12C/RF3MM/*"
```

Comment:
We also ship pytrip to windows users. Making tests with `~` may be a bit confusing for them